itemfactory.py

```diff
@@ -0,0 +1,25 @@
+from engine.worldmap import WorldMap
+from random import randint, choice
+
+def initial_item_populate(new_world):
+
+    possible_items = ['a tree', 'a rock', 'a stream', 'a boot', 'some grass', 'some mushrooms', 'a cloud', 'crumpled paper', 'a fallen tree limb']
+    north_limit = 1
+    south_limit = -1
+    east_limit = 1
+    west_limit = -1
+    lastitem = randint(1,7)
```

Comment:
I suspect you were using lastitem to do something and then you changed your mind. 
This makes your code hard to read when we reach your for loop below.
It looks like you are doing this

For a random number of times (between 1 & 7), 
   add a random item to a random location within these bounds

Can you reduce the amount I have to read and make this code simpler?
In python you can also use '_' for variables when you aren't actually going to use them, so you could:

for _ in range(randint(1,7)):

And also you can get rid of the x_list and y_list local variables, and put the list(range(  right in the call to AddItemAt

I like the use of choice() here, very nice!

Reply:
I thought to get started now on the magic system using magic numbers, is that not a good idea?